docs/docs/40-contributor-guide/10-hacking-on-kargo.md

```diff
@@ -5,10 +5,12 @@ sidebar_label: Hacking on Kargo
 
 # Hacking on Kargo
 
-Kargo is implemented in Go. For maximum productivity in your text editor or IDE,
-it is recommended that you have installed the latest stable releases of Go and
-applicable editor/IDE extensions, however, this is not strictly required to be
-successful.
+Kargo is implemented in Go. To start hacking on Kargo, you can find the complete 
+source code and start exploring the project by visiting our 
+[GitHub Repository](https://github.com/akuity/kargo). For maximum 
+productivity in your text editor or IDE, it is recommended that you have installed
+the latest stable releases of Go and applicable editor/IDE extensions, however, 
+this is not strictly required to be successful.
```

Comment:
I am a little iffy on whether we need this link and whether we need to tell would-be contributors to go check us out on GitHub in the _text_ of the document. There is already a prominent link to GitHub in the doc site's header, anyone likely to contribute probably doesn't need to be told to check out the repo, and anyone who does need to be told that is probably not going to offer us the quality of contributions we are hoping to receive.

Often there is an art to saying more with less and I _think_ this is something we can get by without saying, but I am open to hearing other opinions as to the necessity of this.